src/main/java/pro/trevor/tankgame/rule/impl/version3/ApiV3.java

```diff
@@ -359,6 +359,6 @@ private static class JsonKeys {
         public static final String GOLD = "gold";
         public static final String DONATION = "donation";
         public static final String BOUNTY = "bounty";
-
+        public static final String TIME = "time";
```

Comment:
Can this be timestamp? Or actionSubmissionTime?

Reply:
DONE